### ISSUE TITLE
Mean and std deviation computed on both training set and validation set for CIFAR/SVHN

### ISSUE THREAD

**Issue by ZhuMai** [iboing/CliqueNet#8]
In line 74~76 in dataloader/data_generator.py, mean and standard deviation are computed from both training set and validation set for CIFAR and SVHN.  Should they be computed only over the training set to normalize both training data and validation data in data preprocessing?

**Comment by iboing** [comment]
It makes no significant difference.
